code/modules/overmap/exoplanets/exoplanet_flora.dm

```diff
@@ -40,12 +40,14 @@
 
 /obj/effect/landmark/exoplanet_spawn/plant/do_spawn(var/obj/effect/overmap/visitable/sector/exoplanet/planet)
 	if(LAZYLEN(planet.small_flora_types))
-		new /obj/machinery/portable_atmospherics/hydroponics/soil/invisible(get_turf(src), pick(planet.small_flora_types), TRUE)
+		var/datum/seed/S = pick(planet.big_flora_types)
```

Comment:
if it's picking a path, call the variable `var/seed_path` rather than making it look like a proper datum